src/ontology/mondo-edit.obo

```diff
@@ -566598,7 +566600,7 @@ synonym: "plasminogen deficiency, type 2" EXACT []
 xref: OMIM:217090 {source="MONDO:includedEntryInOMIM"}
 is_a: MONDO:0002242 {source="OMIM:217090"} ! coagulation protein disease
 is_a: MONDO:0003847 {source="OMIM:217090"} ! hereditary disease
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9071 {source="OMIM:217090"} ! PLG
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9071 ! PLG
```

Comment:
This should have been protected https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L27. I will add this back.